shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Val.hs

```diff
@@ -130,45 +131,30 @@ scaledMinDeposit v (Coin mv)
   -- The calculation should represent this equation
   -- minValueParameter / coinUTxOSize = actualMinValue / valueUTxOSize
   -- actualMinValue = (minValueParameter / coinUTxOSize) * valueUTxOSize
-  | otherwise = Coin $ adaPerUTxOByte * (utxoEntrySizeWithoutVal + size v) -- round down
+  | otherwise = Coin $ adaPerUTxOByte * (utxoEntrySizeWithoutVal + size v)
   where
-    -- address hash length is always same as Policy ID length
-    addrHashLen :: Integer
-    addrHashLen = 28
+    -- lengths obtained from tracing on HeapWords of inputs and outputs, plus 6 for Map overhead
+    txoutLen :: Integer
+    txoutLen = 14
```

Comment:
should depend on crypto (same as above comment)
exclude byron addresses from getting MAs